package.json

```diff
@@ -33,6 +33,7 @@
     "@upsetjs/react": "^1.6.2",
     "animate.css": "^4.1.1",
     "bootstrap": "^4.6.1",
+    "classnames": "^2.3.1",
```

Comment:
We're using this despite it not being listed as a dependency. See Typography.tsx